Making Sense Project API/Controllers/CarController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Making_Sense_Project.Model;
+using Making_Sense_Project_API.Model.Repostory;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Making_Sense_Project_API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CarController : ControllerBase
+    {
+        ICarCRUD<Car> _carCRUD;
+        public CarController(ICarCRUD<Car> carCRUD)
+        {
+            _carCRUD = carCRUD;
+        }
+
+        [HttpGet("byId")]
+        public IActionResult Get(int idCar)
+        {
+            Car car = _carCRUD.GetById(idCar);
+            return Ok(car);
+        }
+
+        [HttpGet("All")]
+        public IActionResult GetAll()
+        {
+            var list = _carCRUD.GetAll();
+            return Ok(list);
+        }
+
+        [HttpDelete]
+        public IActionResult Delete(int idCar)
+        {
+            _carCRUD.DeleteById(idCar);
+            return Ok();
+        }
+        [HttpPut("Create")]
```

Comment:
Entiendo que el [verbo HTTP](https://developer.mozilla.org/es/docs/Web/HTTP/Methods) mas adecuado para crear un recurso sería el POST
```suggestion
        [HttpPost("Create")]
```

Reply:
Sii, fallo mío, hice el update y create al mismo tiempo y me confundí ahí, ahora lo corrijo
En cuanto a estructura de diseño está bien o es mejor cambiar algo?
Muchas gracias por tu tiempo Dario.
Saludos!

Reply:
Entiendo que el alcance de este PR es solo migrar del proyecto de consola a un proyecto API, correcto? En caso de que sea así, podrías integrar este PR y realizar nuevos para hacer las funcionalidades restante.
Una sugerencia que puede entrar en este PR es en cuanto al sombre del proyecto, qué te parece algo como `MakingSense.RentCars.Api`  en lugar de `Making Sense Project API` ?

Reply:
Sí exactamente, era solo migrar y que compile con el primer CRUD.
Perfecto, sisi me parece bárbaro, ahora lo cambio y acepto el pr, para continuar con el resto